src/pages/editor/ChannelClaimTableRow.tsx

```diff
@@ -41,67 +56,108 @@ export const ChannelClaimTableRow = ({ claimedChannel, channelType }: ChannelCla
+    },
+    onError: () => dispatch(setNotification({ message: t('feedback.error.delete_channel_claim'), variant: 'error' })),
+  });
 
   return (
-    <TableRow sx={{ bgcolor: 'white' }}>
-      <StyledTableCell aria-live="polite" aria-busy={pendingChannelQuery}>
-        {pendingChannelQuery ? (
-          <Skeleton width={300} />
-        ) : !!channelName ? (
-          <Typography>{channelName}</Typography>
-        ) : (
-          <Typography sx={{ fontStyle: 'italic' }}>{t('common.unknown')}</Typography>
+    <>
+      <TableRow sx={{ bgcolor: 'white' }}>
+        <StyledTableCell aria-live="polite" aria-busy={pendingChannelQuery}>
+          {pendingChannelQuery ? (
+            <Skeleton width={300} />
+          ) : !!channelName ? (
+            <Typography>{channelName}</Typography>
+          ) : (
+            <Typography sx={{ fontStyle: 'italic' }}>{t('common.unknown')}</Typography>
+          )}
+        </StyledTableCell>
+        <StyledTableCell aria-live="polite" aria-busy={organizationQuery.isPending}>
+          {organizationQuery.isPending ? (
+            <Skeleton width={300} />
+          ) : organizationName ? (
+            <Typography>{organizationName}</Typography>
+          ) : (
+            <Typography sx={{ fontStyle: 'italic' }}>{t('common.unknown')}</Typography>
+          )}
+        </StyledTableCell>
+        <StyledTableCell>
+          <StyledChip
+            variant="filled"
+            color="secondary"
+            size="small"
+            sx={{
+              bgcolor: publishingPolicy === 'Everyone' ? 'publishingRequest.main' : 'centralImport.main',
+            }}
+            label={t(`editor.institution.channel_claims.access_policies.${publishingPolicy}`)}
+            icon={
+              publishingPolicy === 'Everyone' ? <LockOpenIcon fontSize="small" /> : <LockOutlineIcon fontSize="small" />
+            }
+          />
+        </StyledTableCell>
+        <StyledTableCell>
+          <StyledChip
+            variant="filled"
+            color="secondary"
+            size="small"
+            sx={{
+              bgcolor: editingPolicy === 'Everyone' ? 'publishingRequest.main' : 'centralImport.main',
+            }}
+            label={t(`editor.institution.channel_claims.access_policies.${editingPolicy}`)}
+            icon={
+              editingPolicy === 'Everyone' ? <LockOpenIcon fontSize="small" /> : <LockOutlineIcon fontSize="small" />
+            }
+          />
+        </StyledTableCell>
+        <StyledTableCell sx={{ display: 'flex', flexWrap: 'wrap', gap: '0.25rem' }}>
+          {claimedChannel.channelClaim.constraint.scope.map((scope) => (
+            <Chip key={scope} variant="filled" color="primary" label={t(`registration.publication_types.${scope}`)} />
+          ))}
+        </StyledTableCell>
+        {isOnSettingsPage && (
+          <>
+            <StyledTableCell>
+              <Tooltip title={t('common.remove')}>
+                <IconButton
```

Comment:
Slett-knappen her ble veldig passiv og vanskelig å se syns jeg? Burde vi ikke her ha noe samme løsning som feks mine registreringer?

![bilde](https://github.com/user-attachments/assets/52d5ff6a-bd18-48f3-8181-e60737942aaa)
![bilde](https://github.com/user-attachments/assets/235f5764-0c7a-409f-9e04-6113c3db1012)

Reply:
Ingen sterke meninger om det. Synes kanksje det er litt rart med `registration`-bakgrunn her da. Noen forslag?

Reply:
Trodde det var en secondary-farge som ble brukt der jeg. Men kan gjøre det på samme måte som for prosjekt da?
![bilde](https://github.com/user-attachments/assets/2f3a8d66-8c8b-4f56-bfa1-ffcf62afac8b)